pkg/users/client.go

```diff
@@ -5,13 +5,14 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"net/http"
```

Comment:
do we not have any custom formatter settings for this SDK?

these changes were made by my editor's format on save that seems to be using the default vs code go formatter